src/transcendental.rs

```diff
@@ -101,20 +101,20 @@ where
 /// neg - bool indicates that operand is negative value.
 pub fn exp<S, D>(operand: S, neg: bool) -> Result<D, ()>
 where
-    S: FixedUnsigned + PartialOrd<D>,
-    D: FixedUnsigned + PartialOrd<S> + From<S>,
+    S: FixedUnsigned + PartialOrd<D> + One,
+    D: FixedUnsigned + PartialOrd<S> + From<S> + One,
 {
     if operand.is_zero() {
-        return Ok(D::from_num(1));
-    };
-    if operand == S::from_num(1) {
+        return Ok(D::one());
+    }
+    if operand == S::one() {
         //TODO: make this as const somewhere
         let e = S::from_str("2.718281828459045235360287471352662497757").map_err(|_| ())?;
```

Comment:
not part of this PR. but i thought this has been already changed to use a constant from fixed crate. what did it disappear?

Reply:
No idea, maybe a garbled merge?

Reply:
Btw. tried to do a quick fix for this in another branch and it seems non-trivial because this function is generic and the code using it seems to somehow not fulfill `FloatConst`? got stuck there

Reply:
it's the same on `main`
https://github.com/galacticcouncil/HydraDX-math/blob/2d7f86ffae242fc855bc833dd22df2d8dc5d03df/src/transcendental.rs#LL112C48-L112C48

Reply:
yes. i just confused it with the other constant. it is ok.